src/batch-client/index.ts

```diff
@@ -764,6 +764,12 @@ export class ZimbraBatchClient {
 					},
 					{
 						_content: 'sendOnBehalfOf'
+					},
+					{
+						_content: 'sendAsDistList'
```

Comment:
isn't this something we need to send through graphql instead of hardcoding here?

Reply:
yes we do that. in that case we should send all this 4 rights from graphql.

Reply:
Done.